src/io/sb3/interfaces.ts

```diff
@@ -0,0 +1,497 @@
+import { OpCode } from "../../OpCode";
+import { KnownBlock, ProcedureBlock } from "../../Block";
+import { TextToSpeechLanguage } from "../../Project";
+import * as BlockInput from "../../BlockInput";
```

Comment:
Can you import this as `_BlockInput` instead? You're defining your own `BlockInput` type and exporting it in this same file, and under some circumstances, the TypeScript compiler will error out if you import something and export something else under the same name.